source/core/slang-type-text-util.cpp

```diff
@@ -197,7 +197,7 @@ struct DebugInfoFormatTable
 
     switch (type)
     {
-        default:    /* fall-through to none */
```

Comment:
Should we use `[[fallthrough]]` or  `; // fall through` ?
`[[fallthrough]]` may not be supported by older compilers.

Reply:
We should use `[[fallthough]]` as a c++17 feature. In this particular case I kept the comment as a reminder to fall through to the specific `None` case.